curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -59,7 +63,7 @@ tests:
       }
     }
     "
-  - text: 'I can add an exercise to any user by posting form data userId(_id), description, duration, and optionally date to /api/exercise/add. If no date supplied it will use current date. App will return the user object with the exercise fields added.'
+  - text: 'You can `POST` to `/api/exercise/add` with form data `userId(_id)`, `description`, `duration`, and optionally `date`. If no date supplied it will use current date. Returned will be the user object with the exercise fields added.'
```

Comment:
```suggestion
  - text: 'You can `POST` to `/api/exercise/add` with form data `userId=_id`, `description`, `duration`, and optionally `date`. If no date supplied it will use current date. Returned will be the user object with the exercise fields added.'
```